views/src/features/Feedbacks/feedbackSlice.js

```diff
@@ -9,11 +9,13 @@ const initialState = {
   error: null,
 };
 
-const createAsyncThunkWithJwt = (type, url, method = "get") => createAsyncThunk(type, async (id, thunkAPI) => {
+const createAsyncThunkWithJwt = (type, url, method = "get") => createAsyncThunk(type, async ( data, thunkAPI) => {
```

Comment:
Quick question on this. I'm guessing `id` is inside the data object correct? I thought the idea was to do `data.id` in the url section.

Reply:
Sorry seeing this late, had to step out for a bit.

I noticed changing the `url` even with conditionals would break things and add onto the api's endpoint url. But I juuust tried with an extra conditional and it seems to not break anything. I will change that back to `url: data?.id ? `${url}${data.id}` : url,` 

Everything I did earlier didn't work but adding that extra `?` after `data` did the trick! 

Do you want me to make another PR alongside the dropdown select changes or keep it separate? I didn't change any of the logic on the feedback form but I did rewrite it using Mantine's stuff to keep everything nice and tidy.

Reply:
Oh its all good. I was just wondering where the id was coming in. As long as it works Its all good. I'm approving. Merge whenever